WebDriverAgentTests/IntegrationTests/FBElementAttributeTests.m

```diff
@@ -48,7 +48,7 @@ - (void)testContainerAccessibilityAttributes
   XCUIElement *inaccessibleButtonElement = self.testedApplication.buttons[@"not_accessible"];
   XCTAssertTrue(inaccessibleButtonElement.exists);
   XCTAssertFalse(inaccessibleButtonElement.fb_isAccessibilityElement);
-  XCTAssertTrue(inaccessibleButtonElement.isWDAccessibilityContainer);
```

Comment:
Here depends on Xcode version...
Xcode 11 was `XCTAssertFalse` in both iOS 12 and 13.